docs/validator/economics.md

```diff
@@ -0,0 +1,134 @@
+---
+id: economics
+title: The Economics of a Validator
+sidebar_label: Understanding Economics
+---
+## Overview
+
+NEAR is a sharded, developer-friendly, proof-of-stake blockchain. It will use a technique called 'dynamic sharding' that splits the network, so that much of the computation is done in parallel. With the increase of users and their transactions, NEAR will dynamically introduce additional shards.
+Therefore, the total number of validators and their assigned shards will change over time, based on the network usage (see [Nightshade documentation pages](../technical/nightshade.md)).
+However, each shard allows a fixed number of *seats*, with validators randomly assigned to one (or more) of them between every `epoch`. The more the shards, the higher the number of available *seats* - so, the more the *seats*, the larger the room for new validators.
+
+As a validator, you need to secure three classes of resources:
+1. NEAR tokens to stake
+2. Nodes infrastructure
+3. Continuous operation
+
+
+## 1. NEAR tokens to stake
+
+NEAR protocol determines validators at the end of each `epoch`, ordering them by the size of their stake. Current validators are re-elected automatically, by *re-staking* their tokens plus the accrued rewards; new validators may replace an existing one if they offer a more significant stake.
```

Comment:
Not sure what you mean by "ordering them by the size of their stake". The order of validators is not determined by their stake.
Also it is imprecise to say that current validator will be re-elected automatically. It is better to say that they will automatically be enrolled in the auction that determines the validators for the future epochs.

Reply:
Happy to review the language to be more specific. I thought that winning proposals are put into an ordered array that we called `validatorAssignments`. If not by the stake, how are they ordered? Aren't the validators elected by the size of their stake - such that higher stakes come first?

Reply:
They are elected by the size of their stake, but the order of elected validators is not determined by their stake. We use the randomness seed from the randomness beacon to shuffle validators.